### DISCUSSION TITLE
Trending always User_Name "Tiktok"

### DISCUSSION THREAD

**Discussion by sw1f1s** [davidteather/TikTok-Api#683] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/10772855/131453580-9c868ba9-5ec8-49f3-a213-64a8de1fa967.png)

Code:
verifyFP ="verify_ksy8gdne_jwSvfCeb_nvz8_4PKj_8MBQ_f2owP06RQnz4"
n_trending = 10

api = TikTokApi.get_instance()

trending_videos = api.by_trending(custom_verifyFP=verifyFP, count=n_trending)
trending_videos = [simple_dict(v) for v in trending_videos]
trending_videos_df = pd.DataFrame(trending_videos)

**Comment by dallinbohn1** [comment]
^^I have the exact same problem. It worked a few weeks ago and then suddenly I can't get the user's username, it's just "tiktok"

**Comment by mhcmh** [comment]
same, and if you try to alter how many results to be return e.g. count=20, it still returns the default 10

**Comment by davidteather** [comment, marked as answer]
This is TokTok's way of detecting bots. This should be fixed in V6
